test/integration/features/story_registry.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Local image to image-registry to deployment
+
+    The user creates a local container image with an app. They then
+    push it to the openshift image-registry in their
+    project/namespace. They deploy and expose the app and check its
+    accessibility.
+
+    Scenario Outline: Start CRC
+        Given executing "crc setup" succeeds
+        When starting CRC with default bundle and hypervisor "<vm-driver>" succeeds
+        Then stdout should contain "CodeReady Containers instance is running"
+        And executing "eval $(crc oc-env)" succeeds
+        And executing "oc login --insecure-skip-tls-verify -u kubeadmin -p 9gZtH-EWZLT-mZNFP-bkKwm https://api.crc.testing:6443" succeeds
```

Comment:
Here the password is hard coded which means this test need to updated everytime when a new release happen :(
Instead `cat <extracted_bundle_path>/kubeconfig`

Reply:
That was an oversight on my part. Thanks!